extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.ts

```diff
@@ -227,7 +250,8 @@ export default class StaticWadoClient extends api.DICOMwebClient {
       return this.compareDateRange(testValue, valueElem.Value[0]);
     }
     const value = valueElem.Value;
-    return this.compareValues(testValue, value);
+
+    return this.compareValues(testValue, value, supportsFuzzyMatching && isPatientName);
```

Comment:
Can you pass in an options object instead of a single key - that way we can add wildcard matching in the future without changing the API.